modules/_labs/select/react/lib/SelectMenu.tsx

```diff
@@ -35,18 +36,28 @@ const fadeOutAnimation = keyframes`
 
 export const menuFadeDuration = 200;
 
-const menuBorderCSS = (error?: ErrorType): CSSObject => {
+const menuBorderCSS = (error?: ErrorType, theme?: EmotionCanvasTheme): CSSObject => {
   let borderColor = inputColors.focusBorder;
   let dividerBorderColor = borderColor;
   let dividerBorderWidth = 1;
-
   if (error === ErrorType.Error) {
-    borderColor = inputColors.error.border;
-    dividerBorderColor = inputColors.error.border;
+    if (theme) {
+      borderColor = theme.canvas.palette.error.main;
+      dividerBorderColor = theme.canvas.palette.error.main;
+    } else {
+      borderColor = inputColors.error.border;
+      dividerBorderColor = inputColors.error.border;
+    }
```

Comment:
i can probably just to a ternary here?

Reply:
I made `theme` a required argument, so no need for a ternary any more.